java/1768-merge-strings-alternately.java

```diff
@@ -0,0 +1,17 @@
+class Solution {
+    public String mergeAlternately(String word1, String word2) {
+        int i = 0, j = 0;
+        StringBuilder res = new StringBuilder();
+
+        while (i < word1.length() && j < word2.length()) {
```

Comment:
Suggestion: use || and check boundaries of your pointers to avoid substring at end of your function.

Reply:
And time and space complexity

Reply:
You can also solve with a single index pointer

Reply:
Addressed all the Suggestions and added the time and space complexity at the end.